libs/wingsdk/src/cloud/counter.ts

```diff
@@ -68,12 +68,23 @@ export interface ICounterClient {
    * @inflight
    */
   inc(amount?: number): Promise<number>;
+
+  /**
+   * Get the current value of the counter
+   * @returns current value
+   * @inflight
+   */
+  peek(): Promise<number>;
 }
 
 /**
- * List of inflight operations available for `Counter`.
+ * List of inflight operations available for `Counter`,
+ * Using this API may introduce race conditions since the value can change between
+ * the time it is read and the time it is used in your code.
```

Comment:
Oops - I think this is in the wrong place (it should be on the `peek` docstring)

Reply:
Sorry, changing now